wavelength-plugins/profile.js

```diff
@@ -343,18 +484,37 @@ exports.commands = {
 			return '<img src="http://flags.fmcdn.net/data/flags/normal/' + ip.country.toLowerCase() + '.png" alt="' + ip.country + '" title="' + ip.country + '" width="20" height="10">';
 		}
 
+		function background(buddy) {
```

Comment:
lets use a variable name that makes more sense like `user`

Reply:
Sure thing